src/main/java/apoc/export/util/TypeExport.java

```diff
@@ -0,0 +1,126 @@
+package apoc.export.util;
+
+import java.lang.reflect.Array;
+import java.util.Iterator;
+
+/**
+ * @author AgileLARUS
+ *
+ * @since 09-06-2017
+ */
+public enum TypeExport {
```

Comment:
Can we make this better? Call it CypherFormat and make it a regular class with subclasses

All the tokens should be part of the implementation and not constructor parameters.